static/src/uu/variables.scss

```diff
@@ -26,9 +26,13 @@ $link-color: $black;
 // Bootstrap list group
 $list-group-active-color: $black;
 
-// Bootstrap bage
+// Bootstrap badge
 $badge-font-size: .65em;
 
+// Group manager table
+$table-active-color: $black;
+$table-active-bg: $primary;
+

```

Comment:
I didn't notice changing the table active color changing the table colors anywhere else but in the group manager, but I could have missed something!